bin/console

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
```

Comment:
Sure.. on the other hand I use it for testing purposes -- it starts ruby repl and loads the module right away.